### PR TITLE
Docs typo fixes 

### PR DESCRIPTION
**Pull request by max-arthurai**

### CODE REVIEW

File: docs/source/custom_scoring.md [hunk 1]
```diff
@@ -9,32 +9,33 @@ In this guide, we will walk through the process of evaluating LLM performance us
...
 
-    def __init__(self, threshold: int = 5):
+    def __init__(self, threshold: int = 3):
         self.threshold = threshold
     
     @staticmethod
     def name() -> str:
         return "trigram_repetition"
-
-    @staticmethod
```
Comment (cheungrowan) [inline docs/source/custom_scoring.md:35]: why is this deleted

Comment (max-arthurai) [reply docs/source/custom_scoring.md:35]: we dont need to override `requires_reference`, its already false by default

Comment (cheungrowan) [reply docs/source/custom_scoring.md:35]: it's true by default: https://github.com/arthur-ai/bench/blob/main/arthur_bench/scoring/scorer.py#L45

Comment (max-arthurai) [reply docs/source/custom_scoring.md:35]: dammit

File: docs/source/concepts.md [hunk 2]
```diff
@@ -4,37 +4,42 @@
...
-Using Bench requires a reference dataset, including information such as:
-- **Inputs** to the LLM. For *Summarization* tasks, this may be the document to be summarized. For *Question & Answering* tasks, this may be the question asked.
-- **Reference Outputs**: these are your baseline outputs. Teams can use either human-labeled ground truth annotations or the outputs from your LLM model in production that you are using Bench to compare against.
-- **Candidate Outputs**: these are the outputs from your new candidate LLM.
-- **Context**: contextual information for Question & Answering tasks.
-
-[//]: # (TODO: are the latter  two part of the reference dataset?)
+Testing LLMs involves preparing the following data for your use case:
+- **Inputs** to the LLM. Depending on the task at hand, these inputs are likely formatted to follow a **prompt template**.
+- **Reference Outputs**: these are your baseline outputs. This would likely be either a ground truth response to the input, or the outputs from a baseline that you are evaluating against.
```
Comment (cheungrowan) [inline docs/source/concepts.md:9]: Can you make clear here that references are not required for all scorers?

File: docs/source/custom_scoring.md [hunk 3]
```diff
@@ -9,23 +9,28 @@ In this guide, we will walk through the process of evaluating LLM performance us
...
 from nltk import trigrams
 from typing import List, Optional
 
-# make sure corpus is downloaded
-nltk.download('punkt')
 
 class TrigramRepetition(Scorer):
 
-    def __init__(self, threshold: int = 5):
+    def __init__(self, threshold: int = 3):
```
Comment (cheungrowan) [inline docs/source/custom_scoring.md:33]: won't changing this to three change the expected outputs at the end?

Comment (max-arthurai) [reply docs/source/custom_scoring.md:33]: yes, the expected outputs at the end are [1.0, 0.0], which this works for

the old example was not scoring the repeated one as a 0.0 because it had less than 5!!

**Review by cheungrowan — APPROVED** [review]
